### ISSUE TITLE
Translation Improvements

### ISSUE THREAD

**Issue by MAGNAT2645** [sapphonie/StAC-tf2#55]
Here's Russian translation: [stac.phrases.txt](https://github.com/sapphonie/StAC-tf2/files/6825026/stac.phrases.txt)
It isn't perfect cuz you can't translate every cheat/exploit name but imo it is better than current (with some fixes).
There's also no DA translations because they were cut by me (this file's from my server).

Also you should remove `{hotpink}[StAC]{white}` from chat translations and move it into macro (or global constant string):
`#define CHAT_PREFIX "{hotpink}[StAC]{white}"`
and then just add it to chat messages, for example:
```c++
Format(pubreason, sizeof(pubreason), CHAT_PREFIX ... " %t", "AimsnapBanAllChat", Cl, aimsnapDetects[Cl]);
```

**Comment by sapphonie** [comment]
Why another Russian translation? I've gotten no complaints re: the current one

**Comment by MAGNAT2645** [comment]
There is at least one wrong translation currently: [here](https://github.com/sapphonie/StAC-tf2/blob/master/translations/stac.phrases.txt#L220).
Also you cant translate most cheat names in Russian so they should be in English, for example:
```
    "AimsnapBanAllChat"
    {
        "#format"   "{1:N},{2:i}"
        "en"        "{hotpink}[StAC]{white} Player {1} was {mediumpurple}aimbotting{white}. Total aimsnap detections: {mediumpurple}{2}{white}. {palegreen}BANNED from server!"
        "ru"        "{hotpink}[StAC]{white} Игрок {1} использовал {mediumpurple}аимботтинг{white}. Всего аимснаппингов обнаружено: {mediumpurple}{2}{white}. {palegreen}Выдана БЛОКИРОВКА на сервере!"
    }
```
should be
```
    "AimsnapBanAllChat"
    {
        "#format"   "{1:N},{2:i}"
        "en"        "{hotpink}[StAC]{white} Player {1} was {mediumpurple}aimbotting{white}. Total aimsnap detections: {mediumpurple}{2}{white}. {palegreen}BANNED from server!"
        "da"        "{hotpink}[StAC]{white} Spiller {1} brugte {mediumpurple}aimbotting{white}. Antal aimsnaps opfanget: {mediumpurple}{2}{white}. {palegreen}BANNED fra serveren!"
        "ru"        "{hotpink}[StAC]{white} Игрок {1} использовал {mediumpurple}AimBot{white}. Всего аимснаппингов обнаружено: {mediumpurple}{2}{white}. {palegreen}ЗАБАНЕН с сервера!"
    }
```

This one just has incorrect translation:
```
    "cmdnumSpikesBanMsg"
    {
        "#format"   "{1:i}"
        "ru"        "[StAC] Имеет блокировку из-за {1} проколов в cmdnum"
    }
```
should be:
```
    "cmdnumSpikesBanMsg"
    {
        "#format"   "{1:i}"
        "ru"        "[StAC] Забанен за {1} скачков cmdnum"
    }
```

**Comment by sapphonie** [comment]
Submit a PR please

**Comment by MAGNAT2645** [comment]
#58

**Comment by sapphonie** [comment]
Closed in #58 when merged into master
